README.md

```diff
@@ -75,36 +82,42 @@ The output will be generated as `output.csv` in the same directory.
 
 The emulator accepts CSV files with the following variables:
 
-### Demographics
-
-| Variable  | Description                     | Notes                                       |
-|-----------|--------------------------------|---------------------------------------------|
-| taxsimid  | Unique identifier              |                                             |
-| year      | Tax year                       |                                             |
-| state     | State code                     |                                             |
-| mstat     | Marital status                 | Only supports: 1 (single), 2 (joint)        |
-| page      | Primary taxpayer age           |                                             |
-| sage      | Spouse age                     |                                             |
-| depx      | Number of dependents           |                                             |
-| age1      | First dependent's age          |                                             |
-| age2      | Second dependent's age         |                                             |
-| ageN      | Nth dependent's age            | Taxsim only allow upto 8 children dependent |
-
-### Income
+### Supported Inputs
+
+| Variable  | Description                                  | Notes                                   |
+|-----------|----------------------------------------------|-----------------------------------------|
+| taxsimid  | Unique identifier                            |                                         |
+| year      | Tax year                                     |                                         |
+| state     | State code                                   |                                         |
+| mstat     | Marital status                               | Only supports: 1 (single), 2 (joint)    |
+| page      | Primary taxpayer age                         |                                         |
+| sage      | Spouse age                                   |                                         |
+| depx      | Number of dependents                         |                                         |
+| age1      | First dependent's age                        |                                         |
+| age2      | Second dependent's age                       |                                         |
+| ageN      | Nth dependent's age                          | Taxsim only allow upto 8 children dependent |
```

Comment:
Child dependents or total dependents?
```suggestion
| ageN      | Nth dependent's age                          | Taxsim only allows up to 8 child dependents |
```

Reply:
in taxsim the **age** vars refer to the child ages based on no of depx provided

Reply:
No, it's not limited to child dependents: https://taxsim.nber.org/taxsim35/
<img width="1699" alt="image" src="https://github.com/user-attachments/assets/59a231d0-10ef-4a23-89e5-f02cb118918d" />

Remove `child` from this description.